src/picknik_ur_gazebo_scan_and_plan_config/config/config.yaml

```diff
@@ -9,3 +9,14 @@ based_on_package: "picknik_ur_gazebo_config"
 optional_feature_params:
   gazebo_world_package_name: "picknik_ur_gazebo_scan_and_plan_config"
   gazebo_world_path: "description/simulation_worlds/scan_and_plan_world.sdf"
+
+objectives:
+  # Override with a new set of waypoints based on the Gazebo world.
+  waypoints_file:
```

Comment:
@sea-bass Since this inherits from the gazebo config, it gets all Objectives there, but overrides the waypoints. I removed the waypoints that no longer made sense, given that we don't have an ISS, boxes or april tags. However, I can't do the same with the Objectives, and I guess there are Objectives that may use some of the waypoints I removed? 

Maybe not something for this PR because of the urgency, but something to keep in mind for a later cleanup. Do you agree?

Reply:
I agree. The "right" thing to do here would be to basically have a bare-bones Gazebo config package, and then have both the space station and scan + plan configs inherit from that, each adding their own unique Objectives.

Let's press on for now.